NEWS.md

```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
```

Comment:
looks like you have a conflict here...?

Reply:
yes i had because you already updated the version number. seems like i forgot to remove everything, should work now